### PR TITLE
tcc : fix arm build

### PR DESCRIPTION
**Pull request by be2020**
* armv7a (androideabi) config fixed
* obsolete patches removed
* add ndk-sysroot dependency (needed to use the compiler)

Note:
- patch was NOT tested under the termux package system really
- however the tcc configuration was tested for armv7a to produce a working package that, when installed on device, can compile tcc again and pass the tests. As in

  git clone git://repo.or.cz/tinycc.git cd tinycc ./configure --cc=tcc make && make test

### CODE REVIEW

File: packages/tcc/build.sh [hunk 1]
```diff
@@ -29,22 +30,12 @@ termux_step_post_get_source() {
...
-	local _ar="$TERMUX_HOST_PLATFORM-ar"
-	cat > "${_ar}" <<-EOF
-		#!$(command -v sh)
-		exec $(command -v $AR) "\$@"
-	EOF
-	chmod 0700 "${_ar}"
-	export PATH="$(pwd):$PATH"
+termux_step_configure() {
+
+        echo "configuring later..."
```
Comment (2096779623) [inline packages/tcc/build.sh:35]: Don't need this line.

**Review by 2096779623 — CHANGES_REQUESTED** [review]

**Comment by xtkoba** [comment]
I won't accept changes that are not tested.

**Comment by xtkoba** [comment]
What is more problematic is that the description is deceptive. This is actually a version bump, not just a fix. I can hardly accept it.

**Comment by xtkoba** [comment]
You should separate the commits of

* fix
* version bump
* refactoring

between one another. Everything in one commit is hard to review.

**Comment by be2020** [comment]
Well, if you find that the patch suggests to remove a lot of lines that according to the commit message were written by yourself, then you are right, but then again you should know better than me what these were meant to do ;)

**Pull request closed without merging**